doc/getting_started/installation_mac.rst

```diff
@@ -108,114 +108,21 @@ To build for the ARM-based Nordic nRF52 Development Kit:
 Setting Up the Toolchain
 ************************
 
-Install tools needed for building the toolchain (if needed):
+In case a toolchain is not available for the board you are using, you can build
+a the toolchain from scratch using crosstool-NG. Follow the steps on the
+crosstool-NG website to prepare your `host
+<http://crosstool-ng.github.io/docs/os-setup/>`_
 
-.. code-block:: console
-
-   brew install gettext help2man mpfr gmp coreutils wget
-   brew tap homebrew/dupes
-   brew install grep --with-default-names
-
-
-To build the toolchain, you will need crosstool-ng version 1.23 or higher.
-Install it by using Homebrew:
-
-.. code-block:: console
-
-   brew install crosstool-ng
-
-Creating a Case-sensitive File System
-=====================================
-
-Building the compiler requires a case-sensitive file system. Therefore, use
-:program:`diskutil` to create an 8 GB blank sparse image making sure you select
-case-sensitive file system (OS X Extended (Case-sensitive, Journaled) and
-mount it.
-
-Alternatively you can use the script below to create the image:
-
-.. code-block:: bash
-
-   #!/bin/bash
-   ImageName=CrossToolNG
-   ImageNameExt=${ImageName}.sparseimage
-   diskutil umount force /Volumes/${ImageName} && true
-   rm -f ${ImageNameExt} && true
-   hdiutil create ${ImageName} -volname ${ImageName} -type SPARSE -size 8g -fs HFSX
-   hdiutil mount ${ImageNameExt}
-   cd /Volumes/$ImageName
-
-When mounted, the file system of the image will be available under
-:file:`/Volumes`. Change to the mounted directory:
-
-.. code-block:: console
-
-   cd /Volumes/CrossToolNG
-   mkdir build
-   cd build
-
-Setting the Toolchain Options
-=============================
-
-In the Zephyr kernel source tree we provide configurations for NIOS-II and
-X86 that can be used to preselect the options needed for building the toolchain.
-
-The configuration files can be found in
-:file:`${ZEPHYR_BASE}/scripts/cross_compiler/`.
-
-Currently the following configurations are provided:
+Follow the steps `here
```

Comment:
Better would be:

    Follow the `Zephyr SDK with Crosstool NG instructions
    <https://github.com/zephyrproject-rtos/sdk-ng/blob/master/README.md>`_ to build ...